statistics/selectivity.go

```diff
@@ -307,8 +329,14 @@ func getMaskAndRanges(ctx sessionctx.Context, exprs []expression.Expression, ran
 	return mask, ranges, false, nil
 }
 
-// getUsableSetsByGreedy will select the indices and pk used for calculate selectivity by greedy algorithm.
-func getUsableSetsByGreedy(nodes []*StatsNode) (newBlocks []*StatsNode) {
+// GetUsableSetsByGreedy will select the indices and pk used for calculate selectivity by greedy algorithm.
+func GetUsableSetsByGreedy(nodes []*StatsNode) (newBlocks []*StatsNode) {
```

Comment:
would it impact the performance?

Reply:
Maybe, because the input order is changed, so the result of greedy search may change.